Geometry_Engine/Query/Height.cs

```diff
@@ -0,0 +1,52 @@
+/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2021, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using BH.oM.Geometry;
+using BH.oM.Reflection.Attributes;
+using System.ComponentModel;
+
+namespace BH.Engine.Geometry
+{
+    public static partial class Query
+    {
+
+        /***************************************************/
+        /****       Public Methods - BoundingBox        ****/
+        /***************************************************/
+
+        [Description("Returns the vertical orthogonal height of a BHoM BoundingBox based on its Z minimum and maximum values.")]
+        [Input("boundingBox", "BHoM BoundingBox to query its height.")]
+        [Output("height", "The height of the BoundingBox based on the difference in Z minimum and maximum values.")]
```

Comment:
The output here should have the Length quantity attribute added. Make sure you add the Using of `BH.oM.Quantities.Attributes` as well.

This should be updated for all methods.

```suggestion
        [Output("height", "The height of the BoundingBox based on the difference in Z minimum and maximum values.", typeof(Length))]
```